### ISSUE TITLE
Document log messages

### ISSUE THREAD

**Issue by ivan-mogilko** [adventuregamestudio/ags-manual#243]
There are script log messages in a following format:
```(room:%d) [%s %d]```
where ags will print error location in square brackets, using a single letter and a number.
https://github.com/adventuregamestudio/ags/blob/12b7f62fce17bb5fdbf4aa5464268b3dcbcebd67/Engine/debug/debug.cpp#L369

May be worth documenting their meaning somewhere. But also have to note that these may be engine-specific and change in future (just in case).

**Comment by ericoporto** [comment]
There was the idea of [GameFeatures](https://github.com/adventuregamestudio/ags-manual/wiki/_GameFeatures) that we didn't fleshed out, perhaps with a link to/from the Log Panel page.

**Comment by ivan-mogilko** [comment]
Yes, I think "engine logging" topic in Engine section would look better, because this is not really a game feature, it's just engine reporting about something.

**Comment by ericoporto** [comment]
Hey, I don't understand what the `?` means in this case. It's not the game script (module ?), Room Script, or the Dialog Script. Which script is it than that would cause the `?` character?

**Comment by ivan-mogilko** [comment]
> Edit: I think G means global script and ? means is any other script module.

Right, and that's kind of weird. I think it's possible that this code was written at the time when custom modules did not exist, so the ? means "undefined behavior". Perhaps we should change it to something, like M or a module's index.
